vmdb/lib/workers/evm_server.rb

```diff
@@ -63,12 +59,18 @@ def start
     end
 
     at_exit {
+      # TODO: should this be called on SOFT ints and SystemExit, not SIGINT?
```

Comment:
@kbrock I think this is the cause of some of the "shutdown_and_exit" messages on the queue.  We run `MiqServer.stop` on SIGINT... something for another day.